lib/components/common/ModalHeightWrapper.tsx

```diff
@@ -1,10 +1,10 @@
 import React from "react";
```

Comment:
I ran the lint command and a couple of files got changes only related with linting. This is one of them